frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/ShareButton/utils.tsx

```diff
@@ -126,7 +128,16 @@ export const loadStateFromQueryParams = ({
   }
 
   // Check for cell types
-  const newFilteredCellTypes = params.get("cellTypes")?.split(delimiter) || [];
+  const newFilteredCellTypes =
+    params
+      .get("cellTypes")
+      ?.split(delimiter)
+      .filter((cellType) => {
+        if (Object.keys(cellTypesByName).indexOf(cellType) !== -1) return true;
```

Comment:
maybe generate `const cellTypeNames = Object.keys(cellTypesByName)` outside of the filter loop?

Reply:
^ @seve not address yet!